docker-compose.yml

```diff
@@ -1,3 +1,20 @@
+x-api-common: &api_common
+  env_file:
+    - .env
+  environment:
+    - KAFKA_HOST=kafka:9092
+    - DATABASE_URL=mysql+asyncmy://root:root_bot_buster@mysql:3306/playerdata
+    - ENV=DEV
+    - POOL_RECYCLE=60
+    - POOL_TIMEOUT=30
+  networks:
+    - botdetector-network
+  depends_on:
+    kafka_setup:
+      condition: service_completed_successfully
+    mysql_setup:
+      condition: service_completed_successfully
```

Comment:
maybe some repetition is good as for example we don't need kafka_host and we may want to toggle environment variable differently, like we can control the port with the environment variable `UVICORN_PORT `